com.github.tenderowl.norka.json

```diff
@@ -0,0 +1,77 @@
+{
+    "app-id": "com.github.tenderowl.norka",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.36",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.tenderowl.norka",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=home",
+        "--filesystem=xdg-run/dconf"
```

Comment:
```suggestion
```

Reply:
Well, seems like I don't need it anymore.